### ISSUE TITLE
WebStorm 2022.3 error at startup

### ISSUE THREAD

**Issue by p-stevenson** [bric3/excalidraw-jetbrains-plugin#104]
Receiving  the follow error message at startup in WebStorm 2022.3

Everything appears to be working as intended, at least I haven't encountered any issues so far, just wanted to make you aware of the issue.

```
Unhandled exception in [StandaloneCoroutine{Cancelling}@4d48f5bc, Dispatchers.Default]

java.lang.AssertionError: Not alive: Lifetime `Anonymous` [Terminated, executing=0, resources=0]
	at com.jetbrains.rd.util.lifetime.RLifetimeKt.assertAlive(RLifetime.kt:524)
	at com.github.bric3.excalidraw.editor.ExcalidrawWebViewController.runJS(ExcalidrawWebViewController.kt:349)
	at com.github.bric3.excalidraw.editor.ExcalidrawWebViewController.saveAsCoroutines(ExcalidrawWebViewController.kt:333)
	at com.github.bric3.excalidraw.editor.ExcalidrawEditor$saveCoroutines$1.invokeSuspend(ExcalidrawEditor.kt:210)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelled}@4d48f5bc, Dispatchers.Default]
```

**Comment by bric3** [comment]
Hi thanks for reporting @p-stevenson,

I don't think this error is problematic, but I'm still having a hard time to prevent this state from happening.

**Comment by bric3** [comment]
Possibly fixed via #20
